lib/rex/proto/dns/resolver.rb

```diff
@@ -115,6 +116,8 @@ def proxies=(prox, timeout_added = 250)
     # @param argument
     # @param type [Fixnum] Type of record to look up
     # @param cls [Fixnum] Class of question to look up
+    #
```

Comment:
Should also document what this can raise via YARD. Think there is ResolverError and NoResponseError